pkg/network/protocols/postgres/operations.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package postgres
+
+import "strings"
+
+// Operation represents a postgres query operation supported by our decoder.
+type Operation uint8
+
+const (
+	UnknownOP Operation = iota
+	SelectOP
+	InsertOP
+	UpdateOP
+	CreateTableOP
+	DropTableOP
+)
+
+// String returns the string representation of the operation.
+func (op Operation) String() string {
+	switch op {
+	case SelectOP:
+		return "SELECT"
+	case InsertOP:
+		return "INSERT"
+	case UpdateOP:
+		return "UPDATE"
+	case CreateTableOP:
+		return "CREATE"
```

Comment:
what about `ALTER` operation

Reply:
We intentionally kept the PR very simple on the types of queries we're processing

> The PR makes the initial ramp up around adding a new protocol to USM, and intentionally keeps the simplicity of tracked queries.
In a series of following PR will add support to more complicated queries, more postgres scenarios, add postgres-TLS support, and more.

Since the PR is complex and touches many files we do the ramping up here, and with a followup pr, and decided ones, we'll add support for more use cases and more queries types